.travis.yml

```diff
@@ -1,14 +1,39 @@
+sudo: true
 language: python
 python:
-  - "2.7"
-  - "3.6"
-
+- 3.6
+cache: pip
+before_install:
+- export BOTO_CONFIG=/dev/null
 install:
-  - pip install pipenv==2018.10.13  # 2018.11.14 is broken. Revisit this once it's fixed.
-  - pipenv install --dev -e ".[s3-backend,redis-backend,trigger,testing]"
+- pip install travispy
+- pip install -e ".[s3-backend,redis-backend,trigger,test]"
 
 script:
-  - pytest --cov
-
+- python -m pytest --cov=driftconfig/ --disable-warnings
+- echo $(git describe --abbrev=0 --tags) > driftconfig/VERSION
 after_success:
- - codecov
+- codecov
+deploy:
+  skip_cleanup: true
+  provider: pypi
+  distributions: sdist bdist_wheel
+  on:
+    tags: true
+    branch:
+    - develop
+    - "/(\\d+\\.)?(\\d+\\.)?(\\*|\\d+)$/"
+  user: __token__
+  password:
+    secure: PUVylYaztJJbWqgSwSNvz0/FOZ7iNln37sSVBi5a/xIQEsk/doOoEWgdtS69JvSdbrea+EWCzs3boe18cEM/5cXAkaI7PM0B0U7VgmkwQSgpqH5A/JVy2kYJM1suzuSo4UxQ4zATvcCV1fd6MqlEu8Tn00vfDjW71jcpZfiAnpH9UKFn5o7j5DmjseuPbjqaeVWDN5e7o6umfFKoiCpDvS1H/lfkuOCQ5BknOsh2R16Whj12jM/bzxp6S0RWUvQeUEGdHAvmNkXZm1NL3E9pB1fWybr3W5mHJYF/QcDY2J93xEL8Vl72GjsQtHgtuxQmLRbFP79F9duwr0vyORAexQ/NUWSWLVlj0qUOeJpAeCJqjZJtArEkniyZ/8qn+hjH9k8hHyXgxh8dckEswCDhDxwMR1GqgbAjRszF/x2KPAqcnL5pdfGALi2Fxzrqp8ZW+LP/0sqPv0KN5h6l3xj0sasZfYNrJfsDRPZe+/IgBF/yLNhmmqvc0Vaazg/v8A4VG6kPUJXRhWbNkzwBfyrJjKHhGAXYAXD2i8ZA9bb3n+Ulmol+TDoO5RS8tvZ7UvmBkmvoI/ujajsAlREQVDNhWAhLK/cnBzPcbqQgAdfOfSn/64TlZraHMgxN9H2Z/7zEIX6qgtmFhiWKa6hqQclBGvOT2nAn2ULspIBEsZSVxlA=
```

Comment:
How secure is this? Who's got the keys? What is it for?

Reply:
Like I mentioned in the other PR, this is encrypted by travis. This is a pypi deployment key.